Makefile

```diff
@@ -565,7 +565,7 @@ style : style-c style-py
 
 CLANG_TIDY ?= clang-tidy
 %.c.tidy : %.c
-	$(CLANG_TIDY) $(TIDY_OPTS) $^ -- $(CPPFLAGS) --std=c99
+	$(CLANG_TIDY) $(TIDY_OPTS) $^ -- $(CPPFLAGS) --std=c99 -I$(CUDA_DIR)/include
```

Comment:
It would be better to use target-specific `CPPFLAGS` up where `-I$(CUDA_DIR)/include` is added to `CPPFLAGS`. We can have a variable to capture both `.o` and `.c.tidy`.

Reply:
This actually came from a commit of mine that is already in master. Do we want to fix it here or in a separate PR?